mypy_django_plugin/django/context.py

```diff
@@ -60,6 +61,13 @@ def noop_class_getitem(cls, key):
         models.QuerySet.__class_getitem__ = classmethod(noop_class_getitem)  # type: ignore
         models.Manager.__class_getitem__ = classmethod(noop_class_getitem)  # type: ignore
 
+        # Define mypy builtins, to not cause NameError during setting up Django.
+        # TODO: temporary/unpatch
```

Comment:
👀

Reply:
The idea here is to unset it afterwards again / only have it during the `"django.setup()"` - but likely/apparently mypy sets/overrides it anyway.
(I've only quickly tested it, and it worked as expected.)

Any suggestion for a test for this?

Reply:
We can create a new test with something like this:

```yaml
-   case: patch_reveal_type_during_django_setup
    main: |
        # main is empty on purpose, because we test the `django.setup()` step:
    installed_apps:
        - myapp
    files:
        -   path: myapp/__init__.py
        -   path: myapp/models.py
            content: |
                # Will be executed during `django.setup()`:
                a = 1
                reveal_type(a)
                reveal_locals()
```

🤔

Reply:
I tried upgrading to current master of django-stubs, and the assert on this line is failing for me, while our current version of django-stubs runs fine.
```
    assert not hasattr(builtins, "reveal_type")
AssertionError
```
Could this assert be too conservative?

Reply:
What do you mean?

Reply:
I'm not quite sure myself hehe.
The precondtion fails when i'm trying to type check locally, so therefore i thought that perhaps this assert might not be needed / rejecting valid django projects.

Reply:
Do you patch your builtins somehow? Maybe some other tools do it for you?
Or is it a problem of `django-stubs` alone?

Anyways, I think you should open a new issue.